numpy/core/tests/test_arrayprint.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
Should just remove it here, tests are not run as scripts. There are two other files where it could be removed

- numpy/distutils/tests/test_misc_util.py
- numpy/fft/tests/test_helper.py